template-contributing-guide.md

```diff
@@ -0,0 +1,341 @@
+
+- Is free (no license required) and easy to use.
+- Is relatively easy to share with collaborators both inside and outside of the Good Docs project (such as with the Write the Docs community).
+- Allows collaborators to give feedback and advice in the form of comments.
+- Tracks comment history for later reference.
+- Has version control capabilities.
+
+That being said, you are not required to work in Google Docs. You can work in your preferred tool (such as Markdown, Microsoft Word, or Gitbooks). However, our experience has shown that Google Docs is the easiest tool for collaborating with other Good Docs Project contributors in the early phases of doc development.
+
+### Recommended research strategies
+
+In our experience, successful templateers usually research their template by:
+
+- **Looking at lots of examples.** Start by searching for examples of that type of document they want to create a template for. The more examples you can look at, the better. While it’s better to review good examples of that type of document, there is actually a lot of value in reviewing bad examples too. Consider keeping a spreadsheet to track which examples you used, what elements each one had in common, and what you thought was effective or ineffective.
+- **Searching for guides, books, blog posts, conference presentations, or videos about best practices.** Search the Internet to find advice, tips, or expert research about how to create that type of document. Consider posting in a forum for resource ideas. For example, asking for helpful guides or insights on a community forum like the Write the Docs Slack workspace could be beneficial. Be mindful of, and respect copyright terms of source material. Do not plagiarize and offer attribution where appropriate.
+- **Reaching out to experts.** When you find people you admire, who have researched your topic already, try reaching out to them. They often have a “how to contact me” webpage. Ask if they’d be okay with their material being used. (They might need to republish under a different copyright.) Invite them to participate in the template working group. They might even be prepared to lead it. If you feel shy about reaching out yourself, your template mentor or senior Good Docs Project member might offer to help.
+- **Collaborating with others in a working group.** Consider joining one of the template-focused working groups to meet regularly with a workshop of writers and discuss research ideas and findings.
+
+
+## Workshop the draft of the template
+
+In this phase, you’ll begin to share your draft with community reviewers and invite feedback.
+Optionally, you might also consider sharing it beyond our community with other technical writing communities such as Write the Docs or beyond.
+The workshop phase is arguably the most crucial and important phase in the template writing process, so your template project might spend the bulk of its time in this phase.

```

Comment:
We might want to rephrase this to avoid the term "to workshop". Perhaps call this the "feedback phase" or the "review phase".

Reply:
I don't like "feedback" because it isn't a verb the way the other phases are. "Review" could work though.

Reply:
Yes, we've reached a decision about this in the US general meeting last week. Resolved: We had a lengthy discussion about it at our last general meeting. The consensus was that the term "workshop," although accurate, was not necessarily understood across all cultures. For that reason, we're going to change the name of this phase to "Get feedback on drafts from the community." There was also some confusion about who could review these drafts. We'll clarify that community reviewers includes anyone who is a member of the Good Docs Project, including members of template writing working groups. For now, we'll keep the requirement to have at least three community reviews are required to move to the next phase. We'll adjust this requirement if we discover it is too difficult.